--- etherape-0.9.18/etherape.desktop.orig	2018-06-03 20:41:11.000000000 +0200
+++ etherape-0.9.18/etherape.desktop	2025-04-25 17:27:23.778519704 +0200
@@ -5,6 +5,7 @@ GenericName[de]=Ein grafischer Netzwerk-
 GenericName[es]=Un monitor gráfico de red
 GenericName[fr]=Un navigateur de réseau graphique
 GenericName[it]=Visualizza il traffico di rete
+GenericName[pl]=Graficzna przeglądarka sieci
 GenericName[ru]=Графический просмотр сети
 GenericName[sv]=En grafisk nätverksbläddrare
 GenericName[tr]=Grafiksel Ağ İzleyici
