diff -Nru etherape-0.5.1/src/Makefile.am etherape-0.5.1.new/src/Makefile.am
--- etherape-0.5.1/src/Makefile.am	Sat Apr 15 21:24:58 2000
+++ etherape-0.5.1.new/src/Makefile.am	Fri Apr 21 10:30:54 2000
@@ -28,7 +28,7 @@
 etherape_LDADD = $(GNOME_LIBDIR) $(GNOMEUI_LIBS) $(INTLLIBS) $(PCAP_LIBS)
 
 install-exec-local:
-	ln -f $(sbindir)/etherape $(sbindir)/interape
+	ln -sf etherape $(DESTDIR)$(sbindir)/interape
 
 uninstall-local:
 	rm -f $(sbindir)/interape
