--- etherape-0.9.4/configure.in.orig	2006-01-11 07:47:42.000000000 +0100
+++ etherape-0.9.4/configure.in	2006-02-01 12:23:25.000000000 +0100
@@ -90,14 +90,25 @@
 # After some kind of change in libc, it seems it is
 # required to #include <resolv.h> for the test to succeed.
 # To go around it I included the test for __res_init
-AC_CHECK_FUNC(res_init, , 
-  AC_CHECK_LIB(bind, res_init, , 
-   AC_CHECK_LIB(resolv, res_init, ,
-    AC_CHECK_LIB(c, __res_init, , [AC_MSG_ERROR([No resolver library found])] ))))
-
-AC_CHECK_FUNC(res_mkquery, , 
-  AC_CHECK_LIB(bind, res_mkquery, , 
-   AC_CHECK_LIB(resolv, res_mkquery, , [AC_MSG_ERROR([No resolver library found])] )))
+AC_CHECK_FUNC(res_init, , [
+  AC_CHECK_FUNC(__res_init, , [
+    AC_CHECK_LIB(resolv, res_init, , [
+      AC_CHECK_LIB(resolv, __res_init, , [
+       AC_CHECK_LIB(bind, res_init, , [AC_MSG_ERROR([No resolver library found])])
+      ])
+    ])
+  ])
+])
+
+AC_CHECK_FUNC(res_mkquery, , [
+  AC_CHECK_FUNC(__res_mkquery, , [
+    AC_CHECK_LIB(resolv, res_mkquery, , [
+      AC_CHECK_LIB(resolv, __res_mkquery, , [
+       AC_CHECK_LIB(bind, res_mkquery, , [AC_MSG_ERROR([No resolver library found])])
+      ])
+    ])
+  ])
+])
 else
 AC_CHECK_FUNC(gethostbyaddr_r, , 
   AC_CHECK_LIB(bind, gethostbyaddr_r, , 
